test/card-test.js

```diff
@@ -17,6 +17,8 @@ describe('cards interactions', function () {
   before(function () {
     this.jsdom = require('jsdom-global')()
     global.$ = global.jQuery = require('jquery');
+	require('jquery-ui-bundle')
```

Comment:
This line is missing a `;`  character at the end. The indentation needs to be inline with line 19 above it.